apps/www/_blog/2024-08-15-log-drains.mdx

```diff
@@ -0,0 +1,81 @@
+---
+title: Introducing Log Drains
+description: Log Drains for exporting product logs is now available under Public Alpha
+author: ziinc
+image: lw12/day-4/log-drains-og.png
+thumb: lw12/day-4/log-drains-thumb.png
+launchweek: '12'
+categories:
+  - launch-week
+  - developers
+  - platform
+tags:
+  - launch-week
+  - o11y
+  - logging
+date: '2024-08-15'
+toc_depth: 3
+---
+
+Today, Supabase is releasing Log Drains for all Team and Enterprise users in Public Alpha.
```

Comment:
```suggestion
Today, Supabase is releasing Log Drains for all Team and Enterprise users.
```

I think we should put " in Public Alpha" at the end under a "Getting started" section or something. These blog posts live on the internet forever, and "Public Alpha" is a snapshot in time